.gitignore

```
@@ -14,6 +14,7 @@ electron/renderer
 electron/models
 electron/docs
 electron/engines
+electron/playwright-report/index.html
```

electron/playwright-report
```suggestion
electron/playwright-report/index.html
```